<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="fileNameLE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="openPB">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ouvrir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitB">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="textRB">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>FichierTexte</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="binRB">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>FichierBinaire</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="consoleTE">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>220</y>
      <width>771</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="affichagePB">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>AffichageNuage3D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="requeteSimplePB">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>RequêteSimple</string>
    </property>
   </widget>
   <widget class="QPushButton" name="requeteOctreePB">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>RequêteOctree</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>770</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBenchOKR">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOubri"/>
    <addaction name="actionClose_all"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuKd_Tree">
     <property name="title">
      <string>Kd-Tree</string>
     </property>
     <addaction name="actionComupe"/>
    </widget>
    <widget class="QMenu" name="menuOctree">
     <property name="title">
      <string>Octree</string>
     </property>
     <addaction name="actionCompute_3"/>
    </widget>
    <widget class="QMenu" name="menuR_Tree">
     <property name="title">
      <string>R-Tree</string>
     </property>
     <addaction name="actionCompute_4"/>
    </widget>
    <addaction name="menuKd_Tree"/>
    <addaction name="menuOctree"/>
    <addaction name="menuR_Tree"/>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>Display</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuBenchOKR"/>
   <addaction name="menuEdit"/>
   <addaction name="menuDisplay"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOubri">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClose_all">
   <property name="text">
    <string>Close all</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionComupe">
   <property name="text">
    <string>Compute</string>
   </property>
  </action>
  <action name="actionCompute">
   <property name="text">
    <string>Compute</string>
   </property>
  </action>
  <action name="actionCompute_2">
   <property name="text">
    <string>Compute</string>
   </property>
  </action>
  <action name="actionCompute_3">
   <property name="text">
    <string>Compute</string>
   </property>
  </action>
  <action name="actionCompute_4">
   <property name="text">
    <string>Compute</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
